js/controller.js

```diff
@@ -1070,6 +1077,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.unsubscribe(OO.EVENTS.ASSET_UPDATED, 'customerUi');
       this.mb.unsubscribe(OO.EVENTS.PLAYBACK_READY, 'customerUi');
       this.mb.unsubscribe(OO.EVENTS.ERROR, "customerUi");
+      this.mb.unsubscribe(OO.EVENTS.SET_EMBED_CODE_AFTER_OOYALA_AD, 'customerUi');
```

Comment:
should `SET_EMBED_CODE_AFTER_OOYALA_AD` be `EMBED_CODE_CHANGED_AFTER_OOYALA_AD` here?